dandi/metadata.py

```diff
@@ -341,45 +345,133 @@ def extract_sex(metadata):
         return ...
 
 
+species_map = [
+    (
+        ["mouse"],
+        "mus",
+        "http://purl.obolibrary.org/obo/NCBITaxon_10090",
+        "Mus musculus - House mouse",
+    ),
+    (
+        ["human"],
+        "homo",
+        "http://purl.obolibrary.org/obo/NCBITaxon_9606",
+        "Homo sapiens - Human",
+    ),
+    (
+        ["norvegicus"],
+        None,
+        "http://purl.obolibrary.org/obo/NCBITaxon_10116",
+        "Rattus norvegicus - Norway rat",
+    ),
+    (
+        ["rattus rattus"],
+        None,
+        "http://purl.obolibrary.org/obo/NCBITaxon_10117",
+        "Rattus rattus - Black rat",
+    ),
+    (
+        ["rat"],
+        None,
+        "http://purl.obolibrary.org/obo/NCBITaxon_10116",
+        "Rattus norvegicus - Norway rat",
+    ),
+    (
+        ["mulatta", "rhesus"],
+        None,
+        "http://purl.obolibrary.org/obo/NCBITaxon_9544",
+        "Macaca mulatta - Rhesus monkey",
+    ),
+    (
+        ["jacchus"],
+        None,
+        "http://purl.obolibrary.org/obo/NCBITaxon_9483",
+        "Callithrix jacchus - Common marmoset",
+    ),
+    (
+        ["melanogaster", "fruit fly"],
+        None,
+        "http://purl.obolibrary.org/obo/NCBITaxon_7227",
+        "Drosophila melanogaster - Fruit fly",
+    ),
+]
+
+
+@lru_cache(maxsize=None)
+def parse_purlobourl(url: str, lookup: ty.Optional[ty.Tuple[str]] = None):
```

Comment:
This declares `lookup` to be an optional tuple of one string.  For an arbitrary number of strings, you need to write `Optional[Tuple[str, ...]]`.

Reply:
```suggestion
def parse_purlobourl(url: str, lookup: ty.Optional[ty.Tuple[str, ...]] = None):
```
?